src/huggingface_hub/_commit_api.py

```diff
@@ -47,6 +51,10 @@
 # See https://github.com/huggingface/huggingface_hub/issues/1503
 FETCH_LFS_BATCH_SIZE = 500
 
+# number of files to upload at a time via xet upload
+# original usage to batch lfs uploads
```

Comment:
This comment doesn't make sense to me (I know this comes from our POC).

I would recommend removing it. The first line is comprehensive enough.